src/ui/components/NetworkMonitor/HttpBody.tsx

```diff
@@ -89,6 +89,7 @@ const HttpBody = ({
       <ReactJson
         style={{ backgroundColor: "none" }}
         theme={theme == "light" ? "rjv-default" : "tube"}
+        name={false}
```

Comment:
Why is this change necessary/reated?

Reply:
Not setting the props results in request body json wrapped inside a `root` key, I guess that is not expected. Request Body json should show up exactly what was in the request. Correct me if I am wrong. Thanks